ratelimit/redis.go

```diff
@@ -96,10 +96,11 @@ func (c *clusterLimitRedis) startSpan(ctx context.Context, spanName string) func
 	span.SetTag("max_hits", c.maxHits)
 	span.SetTag("window", c.window.String())
 
-	return func(failed bool) {
+	return func(allowed, failed bool) {
 		if failed {
 			ext.Error.Set(span, true)
 		}
+		span.SetTag("allowed", allowed)
```

Comment:
Do we want to have it unconditionally or maybe only in case when request is rejected?

Reply:
Yes, I want to have it set like it is proposed.